### PR TITLE
Ability to get real retry state of buffered output

### PR DESCRIPTION
**Pull request by CSharpRU**
Related to #1379 

Will have JSON like this:
```json
{
	// other monitor_agent keys
	"retry": {
		"start":"2016-12-23 01:36:14 +0700",
		"steps":1,
		"next_time":"2016-12-23 01:37:16 +0700"
	}
}
```

### CODE REVIEW

File: lib/fluent/plugin/in_monitor_agent.rb [hunk 1]
```diff
@@ -35,6 +35,7 @@ class MonitorAgentInput < Input
     config_param :tag, :string, default: nil
     config_param :emit_interval, :time, default: 60
     config_param :include_config, :bool, default: true
+    config_param :include_retry, :bool, default: true
```
Comment (CSharpRU) [inline lib/fluent/plugin/in_monitor_agent.rb]: It might be not necessary

File: test/plugin/test_in_monitor_agent.rb [hunk 2]
```diff
@@ -302,7 +302,8 @@ def get(uri, header = {})
         "plugin_category" => "output",
         "plugin_id"       => "null",
         "retry_count"     => 0,
-        "type"            => "null"
+        "type"            => "null",
+        "retry"           => {}
```
Comment (CSharpRU) [inline test/plugin/test_in_monitor_agent.rb]: Need to add test cases for failed flush

**Comment by repeatedly** [comment]
Feature looks good.
But yes, need output failure case.

**Comment by CSharpRU** [comment]
Okay, I'll add it.

**Comment by CSharpRU** [comment]
Test case has been added.

**Comment by repeatedly** [comment]
Good. Thank you!

**Pull request merged**